azext_iot/iothub/_help.py

```diff
@@ -309,3 +309,88 @@ def load_iothub_help():
         type: command
         short-summary: Upload a local file as a device to a pre-configured blob storage container.
     """
+
+    helps[
+        "iot hub state"
+    ] = """
+        type: group
+        short-summary: Manage the state of an IoT Hub.
+    """
+
+    helps[
+        "iot hub state export"
+    ] = """
+        type: command
+        short-summary: Export the state of an IoT Hub to a file.
+        long-summary: The exported state will include hub configurations, edge deployments, device identities and twins, and module
+                      identities and twins.
+        examples:
+        - name: Export the full state of the specified hub to the specified file.
+          text: >
+            az iot hub state export -n {iothub_name} -f {filename}
```

Comment:
IMO file should be optional and is purely convenience. Users can redirect the output of the command to file.

Reply:
Where the output of the command is returned/shown on stdout like most of our commands. Providing `-f` also outputs the content to the file.....but even saying this, its exactly what the `tee` command is used for.

Reply:
Talked about this and wont do - that way import will only accept in a file (and not inline json)